### DISCUSSION TITLE
Can ite() do multiple conditions?

### DISCUSSION THREAD

**Discussion by lfjewett** [hneemann/Digital#814] [Q&A] [answered]
I can't figure out how to get ite() to work properly.  I'm trying to do something like this;

`let d = ite(a=0 && b=0, 0, 1)`

However this doesn't work and throws an error saying 'AND' is not expected.   Is there anyway to evaluate multiple conditions with ite?

**Comment by hneemann** [comment, marked as answer]
Use a single ampersand.

**Reply by lfjewett** [reply]
🤦 Thank you! That worked..
